app/components/HeaderNotifications/index.js

```diff
@@ -37,7 +36,7 @@ const NotificationElement = ({
           styles.notification,
           !notification.read ? styles.unRead : null
         )}
-        onClick={() => markNotification(notification.id)}
+        onClick={() => this.props.markNotification(notification.id)}
```

Comment:
do you still need this line?

Reply:
No, and should I remove markNotification from props? It's unnecessary now anyways

Reply:
Yeah, you can remove it from everywhere (search for it in the codebase) - actions, action types, reducers etc.